### ISSUE TITLE
[Crash] Animania render related client crash when on a server

### ISSUE THREAD

**Issue by HonzaVinCZ** [capnkirok/animaniamod#508]
Hello, I'm using Animania in my modpack and on my server, the problem is that my client crashes sometimes for no reason. Basically I play on the server, game accidentally freezes after random playtime and after few seconds it crashes. From crashlog I found out, it is Animania related. 
I'm not sure if I overlook some incompatibility but I don't think so. Most of the mods are well known.
I just freshly started the server and I got no feedback from my friends yet so I can't say other players has the same problem - I will add an update comment if they get any crashes too. 

Server console outputs only that client disconnected. 

### Steps to Reproduce
Connect to a server, play and client crashes after random time. I also saddled one horse during my hour of playtime - Basically I got a saddle from spawner chest, found a horse, got on him, pressed E after I realised you don't have to tame him, put saddle on  and had a ride. Then I dismounted and did other things like smelting. Then crash accidentally appeared. After launching the game again and connecting to the server, the same crash appeared after about 10 minutes.

### Version of Minecraft, version of Animania, Single Player or Server
MC 1.12.2
animania-1.12.2-base-2.0.3.28
animania-1.12.2-catsdogs-1.0.0.28
animania-1.12.2-farm-1.0.2.28
CraftStudioAPI-universal-1.0.1.95-mc1.12-alpha

### Crash log, latest.log
[latest.log](https://github.com/capnkirok/animaniamod/files/6137811/latest.log)
[crash-2021-03-14_23.25.17-client.txt](https://github.com/capnkirok/animaniamod/files/6137813/crash-2021-03-14_23.25.17-client.txt)
[crash-2021-03-14_23.49.04-client.txt](https://github.com/capnkirok/animaniamod/files/6137814/crash-2021-03-14_23.49.04-client.txt)

**Comment by Tschipp** [comment]
Update dynamic surroundings to the latest BETA version.

**Comment by HonzaVinCZ** [comment]
> Update dynamic surroundings to the latest BETA version.

Updated to latest beta ( DynamicSurroundings-1.12.2-3.6.2.1 )
After removing latest stable build from server and replacing it by beta, server crashes immediately on startup. Even no crash report.
I can see there is some compat with Animania in 3.6.2.0 but any more info why should I use beta? 
When I replaced the beta by the stable, server is running normally again. Obviously it must be caused by Dsurround beta.
![image](https://user-images.githubusercontent.com/29098779/111088692-af3b0980-8528-11eb-9c58-667b54d4e3db.png)

EDIT: Client has exactly the same problem
[latest.log](https://github.com/capnkirok/animaniamod/files/6137971/latest.log)
